app/src/main/java/com/example/nuberjam/ui/main/library/LibraryFragment.kt

```diff
@@ -28,9 +29,21 @@ class LibraryFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        setToolbar()
         setData()
     }
 
+    private fun setToolbar() {
+        val searchButton = binding.appbar.btnSearch
+        val viewTypeButton = binding.appbar.btnViewType
+        val plusButton = binding.appbar.btnPlus
```

Comment:
Who do created action menu onclick button function task?

Reply:
For the grid button, I'll do it on the [MP-370](https://args06.atlassian.net/browse/MP-370). But for the plus button, @RTAgung do it on the [MP-375](https://args06.atlassian.net/browse/MP-375)